addon/appModules/notepad++/autocomplete.py

```diff
@@ -1,11 +1,14 @@
+# coding: utf-8
```

Comment:
is this line necessary?

Reply:
Yes, because on line 15, there are some Non-ASCII characters (⣏ And ⣹).
And since the default encoding in Python 2 is ASCII...

Reply:
Ah. Those could be written as unicode escapes, but I'll fix that if it becomes a problem later.